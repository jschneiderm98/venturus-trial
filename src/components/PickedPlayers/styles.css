.Component {
  box-shadow: 0px 0px 1px 1px #f3ecf2;
  background-image: linear-gradient(#c10077, #6f0087);
  width: 600px;
  height: 250px;
  border-radius: 10px 10px 0 0;
  display: grid;
  grid-template-columns: 40% 20% 40%;
  align-items: center;
}

.Most,
.Least {
  justify-self: center;
}

.Most {
  margin-left: 60px;
}

.PickedText {
  font-size: 20px;
  font-weight: 700;
  color: #f3ecf2;
}

.Player {
  display: flex;
  flex-direction: row;
  width: 100%;
  height: 100%;
}

.Player>span {
  font-size: 16px;
  font-weight: 700;
  color: #f3ecf2;
  margin-top: 15px;
  text-decoration: underline;
  text-underline-position: under
}

.Player span:after {
  content: "\00a0\00a0\00a0\00a0\00a0\00a0";
}

.OutsideCircle {
  border-radius: 50%;
  height: 120px;
  width: 120px;
  display: flex;
  justify-content: center;
  align-items: center;
  margin-top: 10px;
}

.Most>.Player>.OutsideCircle {
  border: 2px dashed #bb6cca
}

.PickCircle {
  --size: 100px;
  background-color: #f3f4f7;
  border-radius: 50%;
  height: var(--size);
  text-align: center;
  width: var(--size);
  padding-right: 10px;
  display: flex;
  justify-content: center;
  align-items: center;
  box-shadow: 0px -5px 20px 0px #6f0087;
}

.Least>.OutsideCircle>.PickCircle {
  border: 2px solid #aa005e;
}

.PickInitials {
  font-size: calc(var(--size) / 2);
  margin-left: 90%;
  color: #93006d;
}

.Middle {
  justify-self: center;
  position: relative;
  width: 1px;
  height: 100%;
  background-color: rgba(248, 241, 247, 0.5);
}

.Middle>div {
  --radius: 70px;
  border-radius: 50%;
  height: var(--radius);
  width: var(--radius);
  border: 1px solid rgba(248, 241, 247, 0.5);
  background-color: transparent;
  position: absolute;
  left: calc(0px - var(--radius) / 2);
  top: calc(50% - var(--radius) / 2);

}