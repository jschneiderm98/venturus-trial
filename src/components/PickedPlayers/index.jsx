import React from 'react';
import './styles.css';

export default function PickedPlayers() {
  return (
    <div className="ComponentBody">
      <div className="Most">
        <span className="PickedText">Most Picked Player</span>
        <div className="Player">
          <div className="OutsideCircle">
            <div className="PickCircle">
              <span className="PickInitials">CR</span>
            </div>
          </div>
          <span>75%</span>
        </div>
      </div>
      <div className="Middle">
        <div />
      </div>
      <div className="Least">
        <span className="PickedText">Least Picked Player</span>
        <div className="Player">
          <div className="OutsideCircle">
            <div className="PickCircle">
              <span className="PickInitials">LM</span>
            </div>
          </div>
          <span>25%</span>
        </div>
      </div>
    </div>
  );
}
