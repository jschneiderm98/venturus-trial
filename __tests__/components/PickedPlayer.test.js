import React from 'react';
import { render } from '@testing-library/react';
import PickedPlayers from '../../src/components/PickedPlayers';

describe('Testing PickedPlayers component', () => {
  test('Should render', () => {
    const tree = render(<PickedPlayers />);

    expect(tree).toMatchSnapshot();
  });
});
