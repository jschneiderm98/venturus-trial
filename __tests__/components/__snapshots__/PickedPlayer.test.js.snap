// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PickedPlayers component Should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ComponentBody"
      >
        <div
          class="Most"
        >
          <span
            class="PickedText"
          >
            Most Picked Player
          </span>
          <div
            class="Player"
          >
            <div
              class="OutsideCircle"
            >
              <div
                class="PickCircle"
              >
                <span
                  class="PickInitials"
                >
                  CR
                </span>
              </div>
            </div>
            <span>
              75%
            </span>
          </div>
        </div>
        <div
          class="Middle"
        >
          <div />
        </div>
        <div
          class="Least"
        >
          <span
            class="PickedText"
          >
            Least Picked Player
          </span>
          <div
            class="Player"
          >
            <div
              class="OutsideCircle"
            >
              <div
                class="PickCircle"
              >
                <span
                  class="PickInitials"
                >
                  LM
                </span>
              </div>
            </div>
            <span>
              25%
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ComponentBody"
    >
      <div
        class="Most"
      >
        <span
          class="PickedText"
        >
          Most Picked Player
        </span>
        <div
          class="Player"
        >
          <div
            class="OutsideCircle"
          >
            <div
              class="PickCircle"
            >
              <span
                class="PickInitials"
              >
                CR
              </span>
            </div>
          </div>
          <span>
            75%
          </span>
        </div>
      </div>
      <div
        class="Middle"
      >
        <div />
      </div>
      <div
        class="Least"
      >
        <span
          class="PickedText"
        >
          Least Picked Player
        </span>
        <div
          class="Player"
        >
          <div
            class="OutsideCircle"
          >
            <div
              class="PickCircle"
            >
              <span
                class="PickInitials"
              >
                LM
              </span>
            </div>
          </div>
          <span>
            25%
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
